daemon/container.go

```diff
@@ -462,6 +469,7 @@ func (container *Container) Kill() error {
 	}
 
 	container.WaitStop(-1 * time.Second)
+	container.LogEvent("kill")
```

Comment:
Its a little hard to follow all of the various kill funcs, but I think you're right. I'll remove this one.